src/utils/ohri-form-helper.ts

```diff
@@ -1,9 +1,9 @@
 import dayjs from 'dayjs';
-import { LayoutType } from '@openmrs/esm-framework';
+import { LayoutType, showToast } from '@openmrs/esm-framework';
 import { fetchConceptNameByUuid } from '../api/api';
 import { ConceptTrue } from '../constants';
 import { EncounterContext } from '../ohri-form-context';
-import { OHRIFormField, OHRIFormPage, OHRIFormSection, SubmissionHandler } from '../api/types';
+import { OHRIFormField, OHRIFormPage, OHRIFormSection, OpenmrsEncounter, SubmissionHandler } from '../api/types';
```

Comment:
You added `showToast` and `OpenmrsEncounter` imports but looks like you don't use them.

Reply:
This is resolved.